File: C/ex14.c
//Entrar via teclado com três valores distintos. Exibir o maior deles.

#include <stdio.h>
#include <stdlib.h>

int main(int argc, char const *argv[])
{
       // Declaração de variáveis
  float a, b, c;
  // Solicita entrada de dados para o usuário e grava o valor digitado em uma variável "a"
  printf("Digite o primeiro valor: ");
   scanf("%f", &a);
  // Solicita entrada de dados para o usuário e grava o valor digitado em uma variável "b"
  printf("Digite o segundo valor: ");
   scanf("%f", &b);
  // Solicita entrada de dados para o usuário e grava o valor digitado em uma variável "c"
  printf("Digite o terceiro valor: ");
   scanf("%f", &c);

   if(a>b){
       if(a>c)
       printf("o primeiro valor %.2f e maior",a);
   }
   else if(b>c){
       printf("o segundo valor %.2f e maior",b);
   }
   else {
       printf("o terceiro valor %.2f e maior",c);
   }
    return 0;
}

File: Python/ex14.py
#Entrar via teclado com três valores distintos. Exibir o maior deles.

vl1 = int(input("Digite o primeiro valor: "))
vl2 = int(input("Digite o segundo valor: "))
vl3 = int(input("Digite o terceiro valor: "))

if (vl1>vl2) and (vl1>vl3):
    print("O maior valor é:", vl1)
if (vl2>vl1) and (vl2>vl3):
    print("O maior valor é:", vl2)
if (vl3>vl1) and (vl3>vl2):
    print("O maior valor é:", vl3)